### ISSUE TITLE
Ask for a a11y review

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#4563]
Update 2/7: The Accessibility team does not have bandwidth to support an audit or review request and will provide recommendations for a third party resource. They may have bandwidth to brainstorm with us about ways to make the VPN a delightful experience for folks using AT. I’m going to close this ticket and open a new ticket for securing a third party resource. 

Update 10/19: We have requested an accessibility review for the VPN client. Once the Accessibility team has completed the review, we will convert this task to an epic and file tickets based on their findings.

We should ask the a11y team to review the VPN product. Here is the documentation:
[https://firefox-source-docs.mozilla.org/bug-mgmt/processes/accessibility-review.html](https://firefox-source-docs.mozilla.org/bug-mgmt/processes/accessibility-review.html)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-2973)
┆Reporter: Andrea Marchesini

**Comment by data-sync-user** [comment]
➤ Andrea Marchesini commented:

We have a lot of a11y issues. Can we make this an epic and link all of the issues here?
